### ISSUE TITLE
3.2.31 causes Column not found: 1054 Unknown column '' in 'field list' 

### ISSUE THREAD

**Issue by nvkodde** [statamic/cms#5108]
### Bug description

Getting the error below in current version when adding collection tag to antlers page. Wasn't sure if this was caused by something else in my existing project so recreated with clean laravel / statamic install and still getting problem.  Rollback to previous version error no longer appears.

SQLSTATE[42S22]: Column not found: 1054 Unknown column '' in 'field list' (SQL: select `` from entries where collection in (articles) and status = published and (json_extract(data, '$."redirect"') is null OR json_type(json_extract(data, '$."redirect"')) = 'NULL') order by json_unquote(json_extract(data, '$."title"')) asc)

### How to reproduce

Steps from a clean install, i'm installing in an existing project and using database for both users and entries (Uuid)

- laravel new
- installed laravel breeze (so could have auth scaffold, will use db for entries and users)
- install statamic
- setup users to use db per docs, and setup entries to be in db per statamic eloquent driver package (Uuid model) -created "pages" collection
- created "articles" collection
- setup layout and default antlers file by taking from coolwritings template and just removed unnecessary stuff (pic link, etc)
- created "home" entry in pages
- set route for pages collection to be root and route as {parent_uri}/{slug}
- set home pages to use home.antlers.html template
- only thing in that page is a collection tag

{{ collection:articles }}

{{ title }}
{{ /collection:articles }}

### Versions

Issue appears at Stamic 3.2.31

Statamic 3.2.31 Solo
Laravel 8.80.0
PHP 8.1.1
statamic/eloquent-driver 0.2.0

### Installation

Existing Laravel app

### Additional details

example project at https://github.com/nvkodde/statamictest which has my steps to recreate documented above.

**Comment by duncanmcclean** [comment]
Hmm, I wonder if this is potentially related to the recent Statamic query builder updates 🤔 

(I haven't actually tried reproducing this yet, just looked at the error message)

**Comment by jasonvarga** [comment]
Can you please provide the stack trace, not just the final error.

**Comment by nvkodde** [comment]
```

#0 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Database/Connection.php(712): ()
--
  | #1 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Database/Connection.php(672): Illuminate\Database\Connection-&gt;runQueryCallback(&#039;select `` from ...&#039;, Array, Object(Closure))
  | #2 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Database/Connection.php(376): Illuminate\Database\Connection-&gt;run(&#039;select `` from ...&#039;, Array, Object(Closure))
  | #3 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(2385): Illuminate\Database\Connection-&gt;select(&#039;select `` from ...&#039;, Array, true)
  | #4 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(2373): Illuminate\Database\Query\Builder-&gt;runSelect()
  | #5 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(2907): Illuminate\Database\Query\Builder-&gt;Illuminate\Database\Query\{closure}()
  | #6 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(2374): Illuminate\Database\Query\Builder-&gt;onceWithColumns(Array, Object(Closure))
  | #7 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(625): Illuminate\Database\Query\Builder-&gt;get(Array)
  | #8 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(609): Illuminate\Database\Eloquent\Builder-&gt;getModels(Array)
  | #9 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Query/EloquentQueryBuilder.php(28): Illuminate\Database\Eloquent\Builder-&gt;get(Array)
  | #10 /Users/nvkodde/Projects/statamictest/vendor/statamic/eloquent-driver/src/Entries/EntryQueryBuilder.php(43): Statamic\Query\EloquentQueryBuilder-&gt;get(Array)
  | #11 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Tags/Concerns/GetsQueryResults.php(25): Statamic\Eloquent\Entries\EntryQueryBuilder-&gt;get()
  | #12 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Tags/Collection/Entries.php(50): Statamic\Tags\Collection\Entries-&gt;results(Object(Statamic\Eloquent\Entries\EntryQueryBuilder))
  | #13 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Tags/Collection/Collection.php(23): Statamic\Tags\Collection\Entries-&gt;get()
  | #14 [internal function]: Statamic\Tags\Collection\Collection-&gt;__call(&#039;articles&#039;, Array)
  | #15 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/View/Antlers/Engine.php(159): call_user_func(Array)
  | #16 [internal function]: Statamic\View\Antlers\Engine::renderTag(Object(Statamic\View\Antlers\Parser), &#039;collection&#039;, Array, &#039;\n    &lt;li&gt;&lt;a hre...&#039;, Array)
  | #17 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/View/Antlers/Parser.php(578): call_user_func_array(Array, Array)
  | #18 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/View/Antlers/Parser.php(177): Statamic\View\Antlers\Parser-&gt;parseCallbackTags(&#039;&lt;ul&gt;\n{{ collect...&#039;, Array)
  | #19 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/View/Antlers/Parser.php(132): Statamic\View\Antlers\Parser-&gt;parse(&#039;&lt;ul&gt;\n{{ collect...&#039;, Array)
  | #20 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/View/Antlers/Engine.php(95): Statamic\View\Antlers\Parser-&gt;parseView(&#039;/Users/nvkodde/...&#039;, &#039;&lt;ul&gt;\n{{ collect...&#039;, Array)
  | #21 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/View/View.php(139): Statamic\View\Antlers\Engine-&gt;get(&#039;/Users/nvkodde/...&#039;, Array)
  | #22 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/View/View.php(122): Illuminate\View\View-&gt;getContents()
  | #23 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/View/View.php(91): Illuminate\View\View-&gt;renderContents()
  | #24 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/View/View.php(94): Illuminate\View\View-&gt;render()
  | #25 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Http/Responses/DataResponse.php(154): Statamic\View\View-&gt;render()
  | #26 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Http/Responses/DataResponse.php(45): Statamic\Http\Responses\DataResponse-&gt;contents()
  | #27 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Structures/Page.php(382): Statamic\Http\Responses\DataResponse-&gt;toResponse(Object(Illuminate\Http\Request))
  | #28 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Routing/Router.php(802): Statamic\Structures\Page-&gt;toResponse(Object(Illuminate\Http\Request))
  | #29 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Routing/Router.php(789): Illuminate\Routing\Router::toResponse(Object(Illuminate\Http\Request), Object(Statamic\Structures\Page))
  | #30 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Routing/Router.php(721): Illuminate\Routing\Router-&gt;prepareResponse(Object(Illuminate\Http\Request), Object(Statamic\Structures\Page))
  | #31 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(128): Illuminate\Routing\Router-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
  | #32 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/StaticCaching/Middleware/Cache.php(34): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #33 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\StaticCaching\Middleware\Cache-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #34 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Http/Middleware/AuthGuard.php(14): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #35 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\Http\Middleware\AuthGuard-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #36 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Http/Middleware/Localize.php(27): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #37 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\Http\Middleware\Localize-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #38 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Http/Middleware/StacheLock.php(29): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #39 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\Http\Middleware\StacheLock-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #40 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(50): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #41 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Routing\Middleware\SubstituteBindings-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #42 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(78): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #43 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #44 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #45 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\View\Middleware\ShareErrorsFromSession-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #46 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(121): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #47 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\Session\Middleware\StartSession-&gt;handleStatefulRequest(Object(Illuminate\Http\Request), Object(Illuminate\Session\Store), Object(Closure))
  | #48 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Session\Middleware\StartSession-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #49 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #50 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #51 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(67): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #52 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Cookie\Middleware\EncryptCookies-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #53 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #54 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Routing/Router.php(723): Illuminate\Pipeline\Pipeline-&gt;then(Object(Closure))
  | #55 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Routing/Router.php(698): Illuminate\Routing\Router-&gt;runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
  | #56 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Routing/Router.php(662): Illuminate\Routing\Router-&gt;runRoute(Object(Illuminate\Http\Request), Object(Illuminate\Routing\Route))
  | #57 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Routing/Router.php(651): Illuminate\Routing\Router-&gt;dispatchToRoute(Object(Illuminate\Http\Request))
  | #58 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(167): Illuminate\Routing\Router-&gt;dispatch(Object(Illuminate\Http\Request))
  | #59 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(128): Illuminate\Foundation\Http\Kernel-&gt;Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
  | #60 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Http/Middleware/DisableFloc.php(18): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #61 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\Http\Middleware\DisableFloc-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #62 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Http/Middleware/CheckMultisite.php(21): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #63 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\Http\Middleware\CheckMultisite-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #64 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Http/Middleware/CheckComposerJsonScripts.php(21): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #65 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\Http\Middleware\CheckComposerJsonScripts-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #66 /Users/nvkodde/Projects/statamictest/vendor/statamic/cms/src/Http/Middleware/PoweredByHeader.php(19): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #67 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\Http\Middleware\PoweredByHeader-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #68 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #69 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php(31): Illuminate\Foundation\Http\Middleware\TransformsRequest-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #70 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #71 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #72 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\Foundation\Http\Middleware\TransformsRequest-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #73 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Foundation\Http\Middleware\TrimStrings-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #74 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #75 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Foundation\Http\Middleware\ValidatePostSize-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #76 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(86): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #77 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Foundation\Http\Middleware\PreventRequestsDuringMaintenance-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #78 /Users/nvkodde/Projects/statamictest/vendor/fruitcake/laravel-cors/src/HandleCors.php(38): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #79 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Fruitcake\Cors\HandleCors-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #80 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Http/Middleware/TrustProxies.php(39): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #81 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Http\Middleware\TrustProxies-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
  | #82 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
  | #83 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(142): Illuminate\Pipeline\Pipeline-&gt;then(Object(Closure))
  | #84 /Users/nvkodde/Projects/statamictest/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(111): Illuminate\Foundation\Http\Kernel-&gt;sendRequestThroughRouter(Object(Illuminate\Http\Request))
  | #85 /Users/nvkodde/Projects/statamictest/public/index.php(52): Illuminate\Foundation\Http\Kernel-&gt;handle(Object(Illuminate\Http\Request))
  | #86 /Users/nvkodde/.composer/vendor/laravel/valet/server.php(234): require(&#039;/Users/nvkodde/...&#039;)
  | #87 {main}


```

**Comment by edalzell** [comment]
@nvkodde can you please show us the version of `statamic\eloquent-driver` you have installed?

**Comment by nvkodde** [comment]
using latest version (from thursday) in my test project ... "statamic/eloquent-driver": "^0.2.0"

**Comment by ryanmitchell** [comment]
Does applying this PR fix the issue for you? https://github.com/statamic/eloquent-driver/pull/27/commits/9f139c541fd2e3106c278d5bd5d2cce720ee173d

**Comment by nvkodde** [comment]
no, different error though

SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '* from `entries` where `collection` in (?) and `status` = ? and (json_extract(`d' at line 1 (SQL: select ``, * from `entries` where `collection` in (articles) and `status` = published and (json_extract(`data`, '$."redirect"') is null OR json_type(json_extract(`data`, '$."redirect"')) = 'NULL') order by json_unquote(json_extract(`data`, '$."title"')) asc)

**Comment by ryanmitchell** [comment]
What MySQL version are you running?

**Comment by nvkodde** [comment]
5.7.36

**Comment by ryanmitchell** [comment]
Are you able to create a GitHub repo with the setup necessary to recreate the issue? I did a fresh install last night but had no luck and I think @edalzell has tried too.

**Comment by nvkodde** [comment]
already did, in my original issue submission https://github.com/nvkodde/statamictest

**Comment by ryanmitchell** [comment]
Sorry, my bad, I missed that initially. 
I've opened a PR with a quick fix for this - https://github.com/statamic/eloquent-driver/pull/40

**Comment by nvkodde** [comment]
This works , appreciate the help!

On Jan 23, 2022, at 2:37 PM, Ryan Mitchell ***@***.***> wrote:

﻿
Sorry, my bad, I missed that initially.
I've opened a PR with a quick fix for this - statamic/eloquent-driver#40

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
